homeassistant/components/nam/strings.json

```diff
@@ -39,15 +39,125 @@
+      "bme280_pressure": {
+        "name": "BME280 pressure"
+      },
+      "bme280_temperature": {
+        "name": "BME280 temperature"
+      },
+      "bmp180_pressure": {
+        "name": "BMP180 pressure"
+      },
+      "bmp180_temperature": {
+        "name": "BMP180 temperature"
+      },
+      "bmp280_pressure": {
+        "name": "BMP280 pressure"
+      },
+      "bmp280_temperature": {
+        "name": "BMP280 temperature"
+      },
+      "heca_humidity": {
+        "name": "HECA humidity"
+      },
+      "heca_temperature": {
+        "name": "HECA temperature"
+      },
+      "mhz14a_carbon_dioxide": {
+        "name": "MH-Z14A carbon dioxide"
+      },
+      "pmsx003_caqi": {
+        "name": "PMSx003 CAQI"
+      },
+      "pmsx003_caqi_level": {
+        "name": "PMSx003 CAQI level",
         "state": {
           "very_low": "Very low",
           "low": "Low",
           "medium": "Medium",
           "high": "High",
           "very_high": "Very high"
         }
+      },
+      "pmsx003_pm1": {
+        "name": "PMSx003 PM1"
+      },
+      "pmsx003_pm10": {
+        "name": "PMSx003 PM10"
+      },
+      "pmsx003_pm25": {
+        "name": "PMSx003 PM2.5"
+      },
+      "sds011_caqi": {
+        "name": "SDS011 CAQI"
+      },
+      "sds011_caqi_level": {
+        "name": "SDS011 CAQI level",
+        "state": {
+          "very_low": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::very_low%]",
+          "low": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::low%]",
+          "medium": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::medium%]",
+          "high": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::high%]",
+          "very_high": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::very_high%]"
+        }
+      },
+      "sds011_pm10": {
+        "name": "SDS011 PM10"
+      },
+      "sds011_pm25": {
+        "name": "SDS011 PM2.5"
+      },
+      "sht3x_humidity": {
+        "name": "SHT3X humidity"
+      },
+      "sht3x_temperature": {
+        "name": "SHT3X temperature"
+      },
+      "sps30_caqi": {
+        "name": "SPS30 CAQI"
+      },
+      "sps30_caqi_level": {
+        "name": "SPS30 CAQI level",
+        "state": {
+          "very_low": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::very_low%]",
+          "low": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::low%]",
+          "medium": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::medium%]",
+          "high": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::high%]",
+          "very_high": "[%key:component::nam::entity::sensor::pmsx003_caqi_level::state::very_high%]"
+        }
+      },
+      "sps30_pm1": {
+        "name": "SPS30 PM1"
+      },
+      "sps30_pm10": {
+        "name": "SPS30 PM10"
+      },
+      "sps30_pm25": {
+        "name": "SPS30 PM2.5"
```

Comment:
I wonder... should I use PM names from these reference keys?
![obraz](https://user-images.githubusercontent.com/478555/229377531-1a12faae-31cd-4601-ae00-aa0157cf546f.png)